frontend/po-files/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+*/
+!.gitignore
```

Comment:
I would move this to the top-level gitignore file

Reply:
All set @spadgett.